modmesh/pilot/_euler1d.py

```diff
@@ -25,22 +25,21 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 
-# This is a hack to make this file runs directly from command line.
-__package__ = "modmesh.app"
-
 import sys
```

Comment:
Group the imports and separate the groups by a single blank line.  The groups are ordered by scope.  The larger the former.